R/PIConfiguration.R

```diff
@@ -31,15 +31,15 @@ PIConfiguration <- R6::R6Class(
         private$.steadyStateTime <- value
       }
     },
-    #' @field printIterationFeedback Boolean. If `TRUE`, the output of the
+    #' @field printCallFeedback Boolean. If `TRUE`, the output of the
     #' residuals calculation will be printed after each iteration.
     #' Default is `FALSE`
-    printIterationFeedback = function(value) {
+    printCallFeedback = function(value) {
```

Comment:
What is the actual difference between "Iteration" and "call"? I think "iteration" sounds less technical, and renaming this would actually be a breaking change (for which I do not see a reason, at least not yet).

Reply:
Rename to "evaluation"